keps/prod-readiness/sig-apps/1847.yaml

```diff
@@ -1,3 +1,3 @@
 kep-number: 1847
-alpha:
+beta:
```

Comment:
Don't change the tag, but add another one. @wojtek-t will you have some time to check this one out?

Reply:
done, kept the reviewer as Wojtek but please LMK if I should switch it to another person.